app/src/main/res/layout/item_list.xml

```diff
@@ -20,7 +22,15 @@
         app:srcCompat="@drawable/ic_drag_vertical_white_24dp"
         tools:ignore="ContentDescription" />
 
+    <View
```

Comment:
@arkon Let me know if there's anything you still see that might not be so clear. I'm pretty sure I explained everything, but I'd be surprised if I didn't miss a thing or two.